### ISSUE TITLE
RbVmomi::Fault: ManagedObjectNotFound error on vm_migrate request

### ISSUE THREAD

**Issue by scambra** [fog/fog-vsphere#75]
I'm trying to migrate vm to different host, using this request:

```ruby
client.vm_migrate 'instance_uuid' => vm.instance_uuid, 'host' => 'hostname'
RbVmomi::Fault: ManagedObjectNotFound: The object 'vim.ManagedEntity:hostname' has already been deleted or has not been completely created
```

Tried migrate vm with vcenter and it works. Also, in recent tasks of vcenter, I see migrate virtual maching task when running vm_migrate request, but I see relocate virtual machine when I migrate vm with vcenter.

**Comment by jrgarcia** [comment]
You mention this is fixed with #74, does this completely fix the issue for you?

**Comment by scambra** [comment]
Yes, it does

**Comment by jrgarcia** [comment]
Closed by #74
